genre.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+
+class Genre < Item
+  attr_accessor :name
+
+  def initialize(name)
+    @id = Random.rand(1...1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
+    item << self
+  end
```

Comment:
Here item is not a list, you can not use "<<" to add a genre to it. Instead, you have to call add_genre method and pass self to it . Kindly, fix that.